blocks/ans-feature-block/features/ans/json.js

```diff
@@ -3,6 +3,7 @@ import getProperties from 'fusion:properties'
 import { resizerKey } from 'fusion:environment'
 import { buildResizerURL } from '@wpmedia/feeds-resizer'
 
+const jmespath = require('jmespath')
```

Comment:
Why does this use a require instead of an import?

Reply:
Updated based on your feedback